packages/cli/templates/default-backend-plugin/src/service/router.ts.hbs

```diff
@@ -0,0 +1,40 @@
+import { errorHandler, loggerToWinstonLogger } from '@backstage/backend-common';
+import { coreServices, createBackendPlugin } from '@backstage/backend-plugin-api';
+import express from 'express';
+import Router from 'express-promise-router';
+import { Logger } from 'winston';
+
+export interface RouterOptions {
+  logger: Logger;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { logger } = options;
+
+  const router = Router();
+  router.use(express.json());
+
+  router.get('/health', (_, response) => {
+    logger.info('PONG!');
+    response.json({ status: 'ok' });
+  });
+  router.use(errorHandler());
+  return router;
+}
+export const {{id}}Plugin = createBackendPlugin({
```

Comment:
Seems like this ought to go in a `plugin.ts.hbs` which then would match up with @davidfestal's suggestion.

Reply:
There is no `plugin.ts.hbs` in `default-backend-plugin`. Do you suggest we create it? What would be the benefit?